language/snippets/src/main/java/com/example/language/Analyze.java

```diff
@@ -107,7 +104,6 @@ public static void analyzeEntitiesText(String text) throws Exception {
       // Print the response
       for (Entity entity : response.getEntitiesList()) {
         System.out.printf("Entity: %s", entity.getName());
-        System.out.printf("Salience: %.3f\n", entity.getSalience());
```

Comment:
nit: is it a removed parameter? Do we want to replace it with smth from v2?

Reply:
Ahhh, this API is still the `AnalyzeEntitySentiment` v1 API. Updated to use the v1 classes instead. Thanks for catching this!